libraries/AP_HAL_ChibiOS/hwdef/QioTekZealotF427/hwdef.dat

```diff
@@ -0,0 +1,229 @@
+# hw definition file for processing by chibios_hwdef.py
+# for QioTekZealotF427 hardware from Qio-tek.com 
+
+# MCU class and specific type
+MCU STM32F4xx STM32F427xx
+
+# board ID for firmware load
+APJ_BOARD_ID 1301
```

Comment:
Needs to be added to board_types.txt

Reply:
OK, thanks for your prompt.

Reply:
I think this should just be 1021 like the bootloader

Reply:
sorry. my mistake, thanks.

Reply:
Sorry. It was my test id, I forgot to modify it.